api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2154,6 +2154,18 @@ message InventoryStatusSummary {
   repeated UpstreamInventoryHello Connected = 1 [(gogoproto.nullable) = false];
 }
 
+// InventoryServiceCountsRequest requests inventory service counts.
+message InventoryServiceCountsRequest {}
+
+// InventoryServiceCounts is the service counts seen in the inventory.
+message InventoryServiceCounts {
+  // ServiceCounts is the count of each service seen in the inventory.
+  map<string, uint64> ServiceCounts = 1 [
+    (gogoproto.nullable) = false,
```

Comment:
You can't have a nullable ProtoBuf `map`; the only effect this has is to _remove_ the `omitempty` option from the json struct tag.